integrations/mistral/tests/test_mistral_chat_generator.py

```diff
@@ -80,18 +80,24 @@ def test_to_dict_default(self, monkeypatch):
         monkeypatch.setenv("MISTRAL_API_KEY", "test-api-key")
         component = MistralChatGenerator()
         data = component.to_dict()
-        assert data == {
-            "type": "haystack_integrations.components.generators.mistral.chat.chat_generator.MistralChatGenerator",
-            "init_parameters": {
-                "api_key": {"env_vars": ["MISTRAL_API_KEY"], "strict": True, "type": "env_var"},
-                "model": "mistral-tiny",
-                "organization": None,
-                "streaming_callback": None,
-                "api_base_url": "https://api.mistral.ai/v1",
-                "generation_kwargs": {},
-            },
+
+        assert (
+            data["type"]
+            == "haystack_integrations.components.generators.mistral.chat.chat_generator.MistralChatGenerator"
+        )
+
+        expected_params = {
+            "api_key": {"env_vars": ["MISTRAL_API_KEY"], "strict": True, "type": "env_var"},
+            "model": "mistral-tiny",
+            "organization": None,
+            "streaming_callback": None,
+            "api_base_url": "https://api.mistral.ai/v1",
+            "generation_kwargs": {},
         }
 
+        for key, value in expected_params.items():
+            assert data["init_parameters"][key] == value
```

Comment:
This avoids the test failure if there are other params in addition to those expected